lib/vrl/tests/tests/expressions/assignment/infallible_mixed.vrl

```diff
@@ -1,4 +1,5 @@
 # result: [{ "ok": 1.0 }, null]
 
-.ok, err = 1 / 1
+foo = 1
+.ok, err = 1 / foo
```

Comment:
This work-around has been fixed in #10340, so I'll update these tests in that PR as well.